### PR TITLE
JBPM-8666: Stunner - Missing the ability to to specify the FEEL expressions

### PR DESCRIPTION
**Pull request by inodeman**
@romartin Can you review this PR, it is for the FEEL expression changes and the readme changes

### CODE REVIEW

**Comment by kie-ci3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by romartin** [comment]
ok to test

File: kie-wb-common-stunner/README.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ The following assumes that the Standalone Showcase will be utilized. Some eviden
 10. Ensure that the *with JavaScript debugger* checkbox is **NOT** checked, as Chrome debugger will be used instead.
 11. Under *Before launch*, click the plus (+) sign.
 12. Select *Run Maven Goal*.
-13. Change the working directory to /[YOUR_DIR_LOCATION]/kie-wb-common/kie-wb-common-stunner/.
+13. Change the working directory to /[YOUR_DIR_LOCATION]/kie-wb-common/kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-standalone/.
```
Comment (romartin) [inline kie-wb-common-stunner/README.md]: thanks for fixing this! :+1:

**Comment by romartin** [comment]
can you please also update the ticket's title with the righ ticket id and description plz?

**Comment by inodeman** [comment]
@romartin I added the test you mentioned and also so that tests would not fail, can you review?

**Comment by wmedvede** [comment]
jenkins retest this

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Thanks for pointing this out @wmedvede !
@inodeman please as commented, do the rebase (and squash) and let us know once it's ready, including the builds ok for testing. 
thx!

**Comment by inodeman** [comment]
Hi @romartin can you re-check this PR? thanks a bunch

**Comment by wmedvede** [comment]
jenkins execute full downstream build

**Comment by hasys** [comment]
### Check list
- [x] ACKs are set
- [x] Affected version is set property
- [x] Is realization aligned to jira description?
- [x] Both Jenkins jobs are green
  * Known selenium instability [BAQE-999](https://issues.jboss.org/browse/BAQE-999) 

### Code review
- [x] Code coverage
- [x] License headers
- [x] Static analyser gates
- [x] Code Style

### Acceptance Criteria
> * Set up FEEL expression for Exclusive gateway and execute both (true and false) branches at runtime 

- [x] Tested with full downstream build
  - [x] Firefox
  - [x] Chrome

### Found issues
>  **Blocker**, **Critical** and **Major** bugs and **Blocker** and **Critical** enhancements should be fixed before merge.

#### Unrelated issues & future improvements
* [JBPM-8670](https://issues.jboss.org/browse/JBPM-8670) - Visual Condition Expression editor doesn't block unknown languages
* [JBPM-8669](https://issues.jboss.org/browse/JBPM-8669) - It is not possible to enter negative value for Integer values during process start
* [JBPM-8668](https://issues.jboss.org/browse/JBPM-8668) - FEEL expression validation

### Used resources
Processes for quick test can be found here: https://github.com/bpmn-tutorials/FEELBPMN

**Comment by romartin** [comment]
Thanks @inodeman , feel free now to close the ticket for this :+1:

**Pull request merged**